### ISSUE TITLE
Boot Loop on 1.7.10

### ISSUE THREAD

**Issue by RaemonDamon** [Kasra-G/ReactorController#17]
I am having issues getting this to run on a 1.7.10 custom modpack. The old pastebin acquisition links don't work, so I've tried fixing the internal CC pastebin program as well as tweaking the pastes from this program to use the version that works, but now it only runs the startup, attempts to calibrate, and then restarts seemingly without actually making the reactorConfig file it's looking for.

**Comment by Gidmark** [comment]
It doesn't seem to work in All the Mods 10 0.51b either with cc-tweaked-1.21.1-forge-1.113.0. Just tries to start and shows a black screen. Running it manually just seems to close it showing "All threads terminated! ..."

**Comment by Kasra-G** [comment]
Thanks for letting me know! I'll take a look and fix if I can reproduce.

**Comment by Kasra-G** [comment]
Investigated, the issue you are describing is caused by os.loadAPI being so old in this version of computercraft that it doesn't work with filenames that end with .lua.

The workaround is to edit the touchpoint file
`edit /usr/apis/touchpoint.lua`
scroll down all the way to the bottom and add this line

```lua
touchpoint = {new = new}
```

Then you have to go to reactorController.lua

`edit reactorController.lua`

and then scroll all the way down to the function

```lua
--Initialize program
local function initialize()
```

and replace
```lua
    os.loadAPI("/usr/apis/touchpoint.lua")
```
with
```lua
    dofile("/usr/apis/touchpoint.lua")
```

Though after some further testing it seems that the graphs are completely broken in this version of computercraft, so not sure if you want to use it for that. I can try to update the code so that it's at least functional out of the box (barring the graphs)

**Comment by Kasra-G** [comment]
The boot loop is fixed by #19 , but since it also removes calibration, Big Reactors is essentially non-functional. If there's enough demand I can revisit fixing support for 1.7.10. Please make a new issue if 1.7.10 support is something you want.
